src/Symfony/Component/DependencyInjection/Argument/AbstractArgument.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
```

Comment:
the code base doesn't use strict types, please remove them from the PR

Reply:
fixed